spec/lex.dd

```diff
@@ -130,8 +130,15 @@ $(GNAME Comment):
     $(GLINK LineComment)
     $(GLINK NestingBlockComment)
 
+$(P In lex language, in order to concatenate, + is used instead of $(OPT) for a block comment.)
```

Comment:
The reference to lex is irrelevant here. I don't really understand what you are trying to convey with this sentence.